src/Renderer/c3DEngine.js

```diff
@@ -15,7 +15,7 @@ import RendererConstant from './RendererConstant';
 
 var instance3DEngine = null;
 
-function c3DEngine(scene, positionCamera, viewerDiv, debugMode, gLDebug) {
+function c3DEngine(scene, positionCamera, targetCamera, viewerDiv, debugMode, gLDebug) {
```

Comment:
Yes, positionCamera and targetCamera are specific GlobeControl
There are removed in PR #239